src/lib/chat/matrix-client.ts

```diff
@@ -417,18 +425,31 @@ export class MatrixClient implements IChatClient {
     };
   };
 
-  private mapUser(userId: string): UserModel {
-    const user = this.matrix.getUser(userId);
-    return {
-      userId: userId,
-      matrixId: userId,
+  private mapUser(matrixId: string, zeroUser?): UserModel {
+    const user = this.matrix.getUser(matrixId);
+    let mappedUser = {
+      userId: matrixId,
+      matrixId,
```

Comment:
I wonder if we would need both userId and matrixId fields/props now

Reply:
I think it's still relevant to know the user's associated matrixId? Because there could be users who are on matrix but NOT on ZERO (yet).